src/styles/main.scss

```
@@ -0,0 +1,68 @@
+$color-minutes: #d15559;
+$color-seconds: #4b9aaa;
+
+@keyframes clock {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@mixin move {
+  animation-name: clock;
+  transform-origin: bottom center;
+  animation-timing-function: linear;
+  animation-iteration-count: infinite;
```

шорткат для animation
```suggestion
  animation: clock linear infinite;
  transform-origin: bottom center;
```